forms-flow-web/src/components/FOI/customComponents/Attachments/index.js

```diff
@@ -485,6 +506,8 @@ const Attachment = React.memo(({indexValue, attachment, handlePopupButtonClick,
     }
   }, [attachment])
 
+  const reclassifyIsDisabled = (attachment.category.includes('>') || attachment.category.toLowerCase().includes('applicant')) ? true : false;
+
```

Comment:
Spoke with Kyle about this. The .includes('>') is the simplest way to check if a file was uploaded by the system. The 'applicant' type is the only other file type without '>' that is tagged by the system. We decided to simply disable reclassifying files tagged with 'applicant'.
The only downside to this is that files manually tagged as 'applicant' cannot be reclassified, they will need to be reuploaded.

Reply:
Spoke with Nick - use the AttachmentCategory enum here to compare to the attachment category value.